notebooks/simple/tab-to-csv.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    for i, p in enumerate(products):
</pre><p>If <code>i</code> is unused in the loop, replace it with <code>_</code>;and maybe a more descriptive name than <code>p</code> would be nice. <code>product</code>?    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Updated</p>